### DISCUSSION TITLE
能否添加一个进程管理功能

### DISCUSSION THREAD

**Discussion by 3154311518** [electerm/electerm#2539] [Ideas]
能否添加一个进程管理功能
实时监视进程的 PID/CPU/内存/读写/宽带/命令行/项目位置

**Comment by zxdong262** [comment]
对于远程ssh链接是有的，点击右下角的信息图标

**Reply by 3154311518** [reply]
好像没有这个功能,右下角是显示系统的信息

就像这样的功能
![1](https://user-images.githubusercontent.com/46740577/175251817-9cb5f946-6daa-4e46-ae03-daabbca40575.png)

**Comment by zxdong262** [comment]
<img width="1108" alt="Screen Shot 2022-06-23 at 22 59 04" src="https://user-images.githubusercontent.com/1641949/175330423-fb6b1ace-dc3f-4105-9181-7eec081b7db3.png">

**Reply by 3154311518** [reply]
![1](https://user-images.githubusercontent.com/46740577/175332457-80a1e768-5927-4b14-8c3f-156eaaae443d.png)
好像没有,昨天刚刚下载的版本

对了好像有个问题,不知道是不是我的电脑问题,还是BUG
命令输入后 下一次输入会有个空格字符

**Reply by 3154311518** [reply]
对了好像有个问题,不知道是不是我的电脑问题,还是BUG
在命令行输入指令后 下一次输入会有个空格字符

**Comment by zxdong262** [comment]
能运行`ps --no-headers -o pid,user,%cpu,size,command ax | sort -b -k3 -r`看看结果吗

**Reply by 3154311518** [reply]
[1.txt](https://github.com/electerm/electerm/files/8968700/1.txt)

**Reply by 3154311518** [reply]
系统Debian

**Reply by 3154311518** [reply]
Debian10

**Comment by zxdong262** [comment]
方便提交个issue吗 https://github.com/electerm/electerm/issues/new/choose
